packages/dds/merge-tree/src/collections/list.ts

```diff
@@ -148,6 +133,41 @@ export class List<T> {
         };
         return iterator;
     }
+
+    private _len: number = 0;
+    private readonly headNode: HeadNode<T> | DataNode<T> = new HeadNode(this);
+    public get length() { return this._len; }
+    public get empty() { return this.headNode._next === this.headNode; }
```

Comment:
```suggestion
    public get empty() { return this._len === 0; }
```
slightly shorter, probably slightly faster (less property accesses)